test/index.ts

```diff
@@ -22,14 +22,16 @@ import * as P from '../src';
 import {paginator, ParsedArguments} from '../src';
 
 const util = {
-  noop: () => {},
+  noop: () => {
+    // do nothing
+  },
 };
 
 class FakeResourceStream extends Transform {
   calledWith: IArguments;
   constructor() {
     super({objectMode: true});
-    this.calledWith = arguments;
+    this.calledWith = arguments; // eslint-disable-line
```

Comment:
got linter error
`Use the rest parameters instead of 'arguments'  prefer-rest-params`